sdks/wasm/Makefile

```diff
@@ -50,9 +50,11 @@ TEST_MCS_DEPS = $(patsubst %,-r:%, $(TEST_DEPS))
 	touch $@
 
 .stamp-v8: .stamp-depot-tools
-	PATH=$(TOP)/sdks/wasm/depot_tools:$$PATH fetch v8
-	cd v8 && tools/dev/v8gen.py -vv x64.release
-	cd v8 && $(TOP)/sdks/wasm/depot_tools/ninja -C out.gn/x64.release
+	rm -rf $(TOP)/sdks/wasm/v8
+	rm -rf $(TOP)/sdks/wasm/.gclient*
+	cd $(TOP)/sdks/wasm/ && PATH=$(TOP)/sdks/wasm/depot_tools:$$PATH fetch v8
```

Comment:
This could be refactored as the following:
```
$(TOP)/sdks/wasm/depot_tools:
	cd $(TOP)/sdks/wasm && git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git

$(TOP)/sdks/wasm/v8: | $(TOP)/sdks/wasm/depot_tools
	cd $(TOP)/sdks/wasm && PATH=$(TOP)/sdks/wasm/depot_tools:$$PATH fetch v8
	cd $(TOP)/sdks/wasm/v8 && tools/dev/v8gen.py -vv x64.release
	cd $(TOP)/sdks/wasm/v8 && $(TOP)/sdks/wasm/depot_tools/ninja -C out.gn/x64.release

.PHONY: toolchain
toolchain: | $(TOP)/sdks/wasm/v8
```

Reply:
That would not handle failures during `git clone` of depot_tools or v8 build failures.